code/controllers/subsystem/ticker.dm

```diff
@@ -322,6 +322,13 @@ SUBSYSTEM_DEF(ticker)
 		else
 			stack_trace("[S] [S.type] found in start landmarks list, which isn't a start landmark!")
 
+	for(var/i in GLOB.human_list)
+		var/mob/living/carbon/human/H = i
+		if(!H.hardcore_survival_score)
+			continue
+		if(H?.mind.special_role)
```

Comment:
```suggestion
		if(H.mind?.special_role)
```

Reply:
But even better, check the player list instead. This one can grow much bigger than that one.

Reply:
bruh :(

Reply:
ninjanomnom would 🗞️ you

Reply:
he's been approving my prs for years!! 😏

Reply:
😡